### PR TITLE
Add document about implementing you own account server

### PR DESCRIPTION
**Pull request by romeovs**

### CODE REVIEW

File: _content/network/account/implement.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+The meaning of rights returned here are:
+
+- `messages:up:r`:  The right to read the uplink messages from devices an application
+- `messages:down:w`: The right to send the downlink messages to devices of an application
+
+The presence of these rights tells the MQTT plugin what type of messages allow
+the client to see or send.
+
+> Important Note: due to the limited right management of most MQTT implementations, make
```
Comment (htdvisser) [inline _content/network/account/implement.md]: Not sure this is true

**Review by htdvisser — APPROVED** [review]
LGTM, 1 small comment

File: _content/network/account/implement.md (hunk 1 above)
Comment (romeovs) [reply _content/network/account/implement.md]: At least conceptually no?

Comment (htdvisser) [reply _content/network/account/implement.md]: I guess most MQTT brokers support read and write rights. 

For example, for Mosquitto:

```c
/*
 * Function: mosquitto_auth_acl_check
 *
 * Called by the broker when topic access must be checked. access will be one
 * of MOSQ_ACL_READ (for subscriptions) or MOSQ_ACL_WRITE (for publish). Return
 * MOSQ_ERR_SUCCESS if access was granted, MOSQ_ERR_ACL_DENIED if access was
 * not granted, or MOSQ_ERR_UNKNOWN for an application specific error.
 */
int mosquitto_auth_acl_check(void *user_data, const char *clientid, const char *username, const char *topic, int access);
```

Similarly, for RabbitMQ:

> RabbitMQ distinguishes between configure, write and read operations on a resource. The configure operations create or destroy resources, or alter their behaviour. The write operations inject messages into a resource. And the read operations retrieve messages from a resource.

Comment (romeovs) [reply _content/network/account/implement.md]: Yes, but the mosquito authentication plugin does not allow us to distinguish between the two.
That's what I'm alluding to. I'll rephrase.

**Pull request merged**